news-explorer/src/components/PopupWithForm/PopupWithForm.js

```diff
@@ -1,26 +1,23 @@
+import Login from "../Login/Login";
+import Register from "../Register/Register";
+import RegistrationSuccessfullyPopup from "../RegistrationSuccessfullyPopup/RegistrationSuccessfullyPopup";
+
 function PopupWithForm(props) {
```

Comment:
`PopupWithForm` should be a universal components for any popup with a form. Please, bring the code back to this component and pass different content through its `children` (like you did in the react project 10-11-14)